src/components/mainPage/UploadSection.jsx

```diff
@@ -57,14 +60,40 @@ const UploadSection = () => {
         .removeEventListener("captchaEvent", captchaListener);
   }, []);
 
+  // changes the visibility of the button depending on the state of files
+  useEffect(() => {
+    if (files.length == 0) {
+      setBtnOpacity(0.4);
+      // console.log(`${files.length} ${btnOpacity}`);
+    } else {
+      setBtnOpacity(1);
+      setUploadAlertState("hidden");
+    }
+  }, [files.length]);
+
+  // prevent button from working if no files are uploaded
+  const handleOnClick = (event) => {
+    if (files.length == 0) {
+      event.preventDefault(event);
```

Comment:
`preventDefault()` doesn't take any arguments afaik

Reply:
what about the other `preventDefault`

https://github.com/Visual-Testing-for-Android-Apps/web/pull/28/files#diff-d2d523e96242a52e5a296514284a8c406de8c9eab11dddcec7465f82e812765fL25

Reply:
Oh yeah, that one shouldn't pass args either